worksheet_A/TerminalHacking.cpp

```diff
@@ -6,6 +6,29 @@
 
 const int wordLength = 5;
 const int numberOfWords = 15;
+int lives = 4;
+
+//Likeness algorithm
+int GetLikeness(std::string guessedWord,std::string secretWord)
+{
+    int result = 0;
+    for (int i = 0; i < secretWord.length(); i++)
+    {
+        if (guessedWord[i] == secretWord[i])
+            result++;
+    }
+    return result;
+}
+//Check if a word is in a list return true or talse.
+bool CheckList(std::set<std::string> list, std::string word)
```

Comment:
`std::set` has a built-in method for this -- see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset . The computational complexity of the built-in method is also better -- O(log n) vs O(n)